### PR TITLE
Multiple tables created

### PR DESCRIPTION
**Pull request by Ali-0111**
- [ ] What animals belong to Melody Pond?
image
========================================================================
- [ ] List of all animals that are pokemon (their type is Pokemon).
image

========================================================================
- [ ] List all owners and their animals, remember to include those that don't own any animal.
image

=========================================================================
- [ ] How many animals are there per species? 
image

=========================================================================
- [ ] List all Digimon owned by Jennifer Orwell. 
image

=========================================================================
- [ ] List all animals owned by Dean Winchester that haven't tried to escape.
image

=========================================================================
- [ ] Who owns the most animals? 
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -81,4 +81,66 @@ SELECT neutered, MAX(escape_attempts) AS Most_Escaped  FROM animals GROUP BY neu
...
+-- Who owns the most animals?
+
+SELECT owners.full_name, count(owners.full_name) AS Number_of_Animals
+FROM animals
+JOIN owners
+ON animals.owner_id = owners.id
+GROUP BY owners.full_name
+ORDER BY Number_of_Animals DESC
+LIMIT 1
+;
```
Comment (iambenkis) [inline queries.sql:146]: - [ ] Keep doing the great work 👏 👏

**Review by iambenkis — CHANGES_REQUESTED** [review]
#### Hi @Ali-0111 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️  Well design Schema
✔️ Queries give expected output 

## Suggested changes

- [ ] Kindly fix the broken link attached to the MIT link to make your README documentation professional 👍 

- [ ] Under the MIT license , the note text is found while it contains text unrelated to you project , kindly delete them , they will make your project documentation even more professional 👍

Capture


_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Ali-0111** [comment]
I deleted extra lines and created MIT LICENSE.md file.

**Review by DeliceLydia — APPROVED** [review]
## Hi @Ali-0111, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- Addressed changes



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**